app/src/ui/add-repository/gitignores.ts

```diff
@@ -1,6 +1,6 @@
 import * as Path from 'path'
-import { readdir, writeFile } from 'fs-extra'
-import { readFile } from 'fs/promises'
+import { readdir } from 'fs/promises'
+import { readFile, writeFile } from 'fs/promises'
```

Comment:
All in one import maybe?
```suggestion
import { readdir, readFile, writeFile } from 'fs/promises'
```

Reply:
Love it!